docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.7"
+services:
+  app-backend:
+    restart: always
+    image: t-learn/backend:latest
+    build: docker/Dockerfile.jvm
```

Comment:
not needed, can't build the image anyway because the compiled application is not there on the server